app/containers/markdown/new/Plain.tsx

```diff
@@ -4,16 +4,17 @@ import { Plain as PlainProps } from '@rocket.chat/message-parser';
 
 import styles from '../styles';
 import { useTheme } from '../../../theme';
-import { themes } from '../../../lib/constants';
 
 interface IPlainProps {
 	value: PlainProps['value'];
+	isLink?: boolean;
 }
 
-const Plain = ({ value }: IPlainProps) => {
-	const { theme } = useTheme();
+const Plain = ({ value, isLink = false }: IPlainProps) => {
+	const { colors } = useTheme();
+	const color = isLink ? { color: colors.actionTintColor } : { color: colors.bodyText };
```

Comment:
I don't get why we need this.
If link is the root element containing children elements with emphasis, it's the link that should have a different color and the children should inherit.

Reply:
Actually, when we set a parameter in a Text Children the param from the children keeps in it, and doesn't receive the params from the parent. Because of that, I needed to do a prop drill.


#### Without the prop drilling
![Screen Shot 2022-10-20 at 19 55 28](https://user-images.githubusercontent.com/47038980/197074290-0bcf172d-33d2-4724-a826-c1e3fa558b61.png)